planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -878,180 +909,186 @@ bool NormalLaneChange::getLaneChangePaths(
 
   candidate_paths->reserve(longitudinal_acc_sampling_values.size() * lateral_acc_sampling_num);
```

Comment:
```suggestion
  candidate_paths->reserve(longitudinal_acc_sampling_values.size() * lateral_acc_sampling_num * prepare_durations.size());
```
reserve size also should be changed?